protobuf/src/test/java/io/grpc/protobuf/StatusProtoTest.java

```diff
@@ -176,6 +176,15 @@ public void fromThrowable_shouldReturnNullIfNoEmbeddedStatus() {
     assertNull(StatusProto.fromThrowable(nestedSe));
   }
 
+  @Test
+  public void toStatusExceptionWithMetadataAndCause_shouldCaptureCause() {
+    RuntimeException exc = new RuntimeException("This is a test exception.");
+    StatusException se = StatusProto.toStatusException(STATUS_PROTO, new Metadata(), exc);
+
+    assertNotNull(se.getCause());
+    assertEquals(se.getCause(), exc);
```

Comment:
`assertSame(exc, se.getCause())` - in JUnit the expected value is passed first.
`assertNotNull` above is redundant.